### PR TITLE
Added 'Hosting Uno Platform WASM on AWS Amplify, by David Onoh' and updated the introduction for improved clarity. Please feel free to review and edit as necessary.

### LINKED ISSUE #10768: Doc needed - Hosting Uno-Wasm applications on Amazon AWS
Hosting Uno Platform Wasm applications is possible on Amazon AWS, but documentation isn't provided. Can we please develop a document similar to those existing for Azure, Nginx and Apache to help community deploy to Azure AWS. 

Similar pages
[Hosting a WebAssembly App](https://platform.uno/docs/articles/how-to-host-a-webassembly-app.html)

### PR DESCRIPTION
**Pull request by davidconoh**
Added 'Hosting Uno Platform WASM on AWS Amplify, by David Onoh' and updated the introduction for improved clarity. Please feel free to review and edit as necessary.

GitHub Issue (If applicable): closes #10768

## PR Type
`docs:` improves Uno's documentation and usability in the community.

## What kind of change does this PR introduce?

Other... Please describe: Adding community tutorial to help bring Uno Platform to AWS

## What is the current behavior?

.[Doc needed - Hosting Uno-Wasm applications on Amazon AWS].(https://github.com/unoplatform/uno/issues/10768)

## What is the new behavior?

.[Hosting Uno Platform WASM on AWS Amplify].(https://reads.davidconoh.me/hosting-uno-platform-wasm-on-aws-amplify)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes/features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes/features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable): #10768

### CODE REVIEW

File: doc/articles/guides/community-tutorials.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ## Community Tutorials
 
-This page lists links our community provided tutorials.
+This page lists links to our community provided tutorials.
 
 > [!NOTE]
-> Please use this page to submit links to your blogs, tutorials and other Uno Platform advocacy as part of Hacktoberfest 2022.
+> Please use this page to submit links to your blogs, articles, and other resources as contribution to Uno Platform advocacy for Hacktoberfest 2022.
```
Comment (Youssef1313) [inline doc/articles/guides/community-tutorials.md:6]: Should we remove this note now that Hacktoberfest 2022 already ended?

**Review by Youssef1313 — COMMENTED** [review]
@davidconoh You need to follow conventional commits for CI to pass

File: doc/articles/guides/community-tutorials.md (hunk 1 above)
Comment (davidconoh) [reply doc/articles/guides/community-tutorials.md:6]: I don't think I'm the best person to decide how this goes. I think the team would help us out here.

**Comment by davidconoh** [comment]
@Youssef1313 This the my first time contributing to the Uno Platform Uno repo. I have a Conventional Commit structure that looks like this:
```
`docs:` a commit that adds or improves Uno's documentation.

Adds support for deploying Uno Platform to AWS Amplify and updates the document.
```
I have no idea where to fix it. Maybe you can come in handy :)

**Comment by Youssef1313** [comment]
I tried to fix it for you, but since `master` in your fork is protected, I couldn't force push to it.

image

You should be able to do it like the following

```
git reset --soft HEAD~1
git commit -am "docs: Added 'Hosting Uno Platform WASM on AWS Amplify, by David Onoh'"
git push -f
```

You might face the same error as me, so you'll need to make your master non-protected, or, better to not actually commit to master in your fork and make the change in a separate branch.

**Comment by Youssef1313** [comment]
You are pushing new commits, but the older commits remain with wrong commit messages so this will still be failing. You need to squash the commits (easiest way with `git reset --soft` and re-committing and then force pushing)

or alternatively, start with a new branch (other than `master`) in a new PR.

**Comment by Youssef1313** [comment]
Oh I see you already opened another PR. Closing this one in favor of https://github.com/unoplatform/uno/pull/11654

**Comment by davidconoh** [comment]
This PR was closed because it contained ambiguous information relating to the Issue. The solution has been transferred to: [docs: Added 'Hosting Uno Platform WASM on AWS Amplify, by David Onoh' #11654](https://github.com/unoplatform/uno/pull/11654).

**Pull request closed without merging**